UnityProject/Assets/Scripts/Systems/Research/Objects/ResearchServer.cs

```diff
@@ -92,6 +130,30 @@ private IEnumerator TrickleResources()
 		public bool MultiMaster => true;
 		int IMultitoolMasterable.MaxDistance => int.MaxValue;
 
+		public bool WillInteract(HandApply interaction, NetworkSide side)
+		{
+			return false; //Techweb Server has no use for now so leave it like this until we're done work on it
```

Comment:
don't you want to do the interaction described below?

Reply:
yeah i think that should be removed currently because it serves no pourpse ever since i removed the entire thing